scheduler/src/main/resources/application.properties

```diff
@@ -1,28 +1,26 @@
-#datasource for application and quartz
-
-spring.datasource.driverClassName=org.postgresql.Driver
-spring.datasource.url=jdbc:postgresql://127.0.0.1/autoscaler
-spring.datasource.username=postgres
-spring.datasource.password=postgres
-
-#quartz job
-scalingenginejob.reschedule.interval.millisecond=10000
-scalingenginejob.reschedule.maxcount=6
-scalingengine.notification.reschedule.maxcount=3
+# datasource for application and quartz
```

Comment:
Git will always remember it but if we don't need it, it is more confusing to have it.